packages/react-scripts/template/README.md

```diff
@@ -846,20 +846,21 @@ You don’t have to use [React Bootstrap](https://react-bootstrap.github.io) tog
 Install React Bootstrap and Bootstrap from npm. React Bootstrap does not include Bootstrap CSS so this needs to be installed as well:
 
 ```sh
-npm install --save react-bootstrap bootstrap@3
+npm install bootstrap@4
```

Comment:
This contradicts the line above, which says "Install React Bootstrap and Bootstrap from npm" (not just install Bootstrap). To use Bootstrap 4 (which is not yet supported by React Bootstrap), [reactstrap](https://reactstrap.github.io/) is a popular option.

Reply:
It would be great to recommend Reactstrap.